src/liballoc/arc.rs

```diff
@@ -332,6 +332,15 @@ impl<T: ?Sized> Deref for Arc<T> {
     }
 }
 
+#[stable(feature = "rc_arc_as_ref", since = "1.2.0")]
```

Comment:
Aha! Yeah I thought I grepped to check but I must have misgrepped, I see that in the codebase now.